use_cases/eluc/data/eluc_data.py

```diff
@@ -58,6 +60,21 @@ def decode_as_df(self, df: pd.DataFrame) -> pd.DataFrame:
                 new_df[col] = new_df[col] * (max_val - min_val) + min_val
         return new_df
 
+    def save_fields(self, path: Path):
+        """
+        Saves the fields to a JSON file.
+        """
+        with open(path, "w", encoding="utf-8") as file:
+            json.dump(self.fields, file, indent=4)
+
+    @classmethod
+    def from_json(cls, path: Path):
+        """
+        Loads the fields from a JSON file.
+        """
+        with open(path, "r", encoding="utf-8") as file:
+            fields = json.load(file)
+        return cls(fields)
```

Comment:
Allows us to load an encoder from a fields file. This enables us to use a pretrained one in the demo.

Reply:
It's important we use the same encoder as the one that was used for training indeed.